product_docs/docs/pgd/5/routing/administering.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Administering PGD Proxy
+navTitle: Administering
+---
+
+## Switching the write leader
+
+Switching the write leader is a manual operation that you can perform to change the node that's the write leader. 
+It can be useful when you want to perform maintenance on the current write leader node or when you want to change the write leader for any other reason.
+When changing write leader, there are two modes, `strict` and `fast`. 
+In `strict` mode, the lag is checked before switching the write leader. It will wait until the lag is less than `route_writer_max_lag` before starting the switchover. This is the default.
+In `fast` mode, the write leader is switched immediately.
+You can also set a timeout parameter to specify the time to wait for the switchover to complete.
+
+!!! Note
```

Comment:
This note is being displayed as a code snippet. Maybe an additional space after the closing !!! will help? 

![Screenshot 2024-05-16 at 11 35 38](https://github.com/EnterpriseDB/docs/assets/98893586/4e62a8ca-bc95-4f49-b257-9f6bfbe465eb)